### PR TITLE
ION-Roller: Immutable software deployment service for AWS

### PR DESCRIPTION
**Pull request by nataliab**

### CODE REVIEW

File: _posts/2015-09-16-ion-roller-immutable-software-deployment-service-for-aws.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: ION-Roller: Immutable software deployment service for AWS
+author: Gary Coady, Natalia Bartol
+date: '2015-09-16'
+categories: 'tech blog'
```
Comment (shellscape) [inline _posts/2015-09-16-ion-roller-immutable-software-deployment-service-for-aws.md]: `tech` category is correct, but `blog` category isn't, as this isn't a post *about* the blog. I'd use `aws` or `ionroller` if you're planning on doing followup posts about ionroller, or if you think there's a real possibility of more posts.

File: _posts/2015-09-16-ion-roller-immutable-software-deployment-service-for-aws.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: ION-Roller: Immutable software deployment service for AWS
+author: Gary Coady, Natalia Bartol
```
Comment (shellscape) [inline _posts/2015-09-16-ion-roller-immutable-software-deployment-service-for-aws.md]: I'm not sure if the author field supports multiples. You may have to update again if it comes out funky.

File: _posts/2015-09-16-ion-roller-immutable-software-deployment-service-for-aws.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+date: '2015-09-16'
+categories: 'tech blog'
+tags:
+- aws
+- docker
+- deployment
+---
+
+Gilt has always been at the forefront of software development, and has been leading the new wave of microservices, which divides the software platform into many individual services that try to do one thing well. Along with the large increase in the number of services, comes the extra complexity of service management.
```
Comment (krschultz) [inline _posts/2015-09-16-ion-roller-immutable-software-deployment-service-for-aws.md]: This first paragraph comes off a little boasty. Maybe 'Gilt has been at the forefront of the wave of microservice architecture. With the advantage of many individual services that do one thing well comes increased complexity of service management.'

**Comment by nataliab** [comment]
I pushed updated version, please take a look!

**Pull request merged**